jbpm-wb-kie-server/jbpm-wb-kie-server-api/src/main/java/org/jbpm/workbench/ks/utils/KieServerUtils.java

```diff
@@ -44,7 +44,8 @@ public class KieServerUtils {
     private static final Logger LOGGER = LoggerFactory.getLogger(KieServerUtils.class);
 
     private static boolean KIE_SERVER_FORM_RENDERER = Boolean.parseBoolean(System.getProperty("org.jbpm.wb.forms.renderer.ext", "false"));
-    
+    private static Long KIE_SERVER_CLIENT_TIMEOUT = Long.parseLong(System.getProperty("org.jbpm.wb.kie.server.client.timeout", "60000"));
```

Comment:
@bxf12315 The change looks good, but I guess, this property needs to be defined at KieServerConstants to have the available properties centralized or maybe document this configuration property to make this visible.

Reply:
KieServerConstants using in kie-server side, the constants in jbpm-wb side.

Reply:
@bxf12315 makes sense not to have it in KieServerConstants, my only concern is about how to make this property 'visible', not just adding the string in a class.

Reply:
yes, I agree with you. I think we need Constants class like KieServerConstants.